prac_09/taxi_simulator.py

```diff
@@ -0,0 +1,85 @@
+"""
+CP1404 Practical 9
+Taxi Simulator exercise.
+"""
+from prac_09.silver_service_taxi import SilverServiceTaxi
+from prac_09.taxi import Taxi
+
+MENU = "q)uit, c)hoose taxi, d)rive\n>>> "
+
+
+def main():
+    """Taxi simulator program - choose a taxi, drive, gain bill."""
+
+    total_bill = 0.0
+    current_bill = 0.0
+    current_taxi = None
+    taxis = [
+        Taxi("Prius", 100),
+        SilverServiceTaxi("Limo", 100, 2),
+        SilverServiceTaxi("Hummer", 200, 4)
+    ]
+
+    print("Let's drive!")
+    menu_choice = input(MENU).lower()
+    while menu_choice != "q":
+
+        if menu_choice == "c":
+            print("Taxis available:")
+            display_taxis(taxis)
+            current_taxi = get_taxi_choice(taxis)
+
+        elif menu_choice == "d":
+            if not current_taxi:  # Check if current_taxi is empty.
+                print("You need to choose a taxi before you can drive")
+            else:
+                drive_taxi(current_taxi)
+                current_bill = current_taxi.get_fare()
+                total_bill += current_bill
```

Comment:
Did you consider an easier way to add to you total bill like `total_bill += current_taxi,get_fare()`?